backend/core/views.py

```
@@ -444,7 +444,8 @@ def provider(self, request):
 
     @action(detail=False, name="Get threats count")
     def threats_count(self, request):
-        return Response({"results": threats_count_per_name(request.user)})
+        folder_id = request.query_params.get("folder", None)
+        return Response({"results": threats_count_per_name(request.user, folder_id)})
```

Ruff (0.8.2)

`threats_count_per_name` may be undefined, or defined from star imports

(F405)